run_benchmark.sh

```diff
@@ -90,10 +90,10 @@ for PREFETCH_SET in "${PREFETCHER_SETTINGS[@]}"; do
   done
 done
 
-if [ ! $IS_POWER ]; then
-  benchmark/prefetching_intel -e  
-else
+if [ "$IS_POWER" = true ]; then
```

Comment:
if $IS_POWER; then

This should be used to be consistent (Is this equivalent?)

Reply:
No, this would not work (see <https://unix.stackexchange.com/a/105946>). 
> So, basically [ $foo ] will be true if $foo is non-empty. Not true or false, just non-empty. [ ! $foo ] is true if $foo is empty or undefined.

Reply:
Bash sucks. But then you should update all is_power checks as well.

Reply:
Actually there is a difference between:
if $IS_POWER; then --> WORKS (but not as explicit as if [ "$IS_POWER" = true ])
and
if [ $IS_POWER ]; then -->WRONG